helix-core/src/main/java/org/apache/helix/controller/dataproviders/BaseControllerDataProvider.java

```diff
@@ -96,13 +99,15 @@
 
   // Special caches
   private CurrentStateCache _currentStateCache;
+  private CustomizedStateCache _customizedStateCache;
   private InstanceMessagesCache _instanceMessagesCache;
 
   // Other miscellaneous caches
   private Map<String, Long> _instanceOfflineTimeMap;
   private Map<String, Map<String, String>> _idealStateRuleMap;
   private Map<String, Map<String, Set<String>>> _disabledInstanceForPartitionMap = new HashMap<>();
   private Set<String> _disabledInstanceSet = new HashSet<>();
+  private Set<String> _aggregationEnabledTypes = new HashSet<>();
```

Comment:
I've removed it, it used to be used by customized state cache, but now we've changed that cache and access the customized state config directly in the cache. So we do not need this variable.

Reply:
I added the aggregationEnableType back, as it will break some test. The main reason is that if we put the accessing to ZK inside customized state cache, it will break some assumption in the mock zk Helix accessor about refresh logic, and the calculation for "config" type would be wrong.